libs/vm_monitor/src/dispatcher.rs

```diff
@@ -118,7 +119,12 @@ impl Dispatcher {
     /// serialize the wrong thing and send it, since `self.sink.send` will take
     /// any string.
     pub async fn send(&mut self, message: OutboundMsg) -> anyhow::Result<()> {
-        info!(?message, "sending message");
+        if matches!(&message.inner, HealthCheck { .. }) {
```

Comment:
For consistency, I would recommend just importing `OutboundMsgKind` and then use `OutboundMsgKind::HealthCheck` here (that way, it gets added to the `use crate::protocol::{ ... }`)